helm/reporting-agent-chart/templates/_helpers.tpl

```diff
@@ -0,0 +1,25 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
what is that?

Reply:
Is to enable filetype in vim, and it use mustache templating. I can remove it

Reply:
no just curious